### PR TITLE
filter slug

### LINKED ISSUE #68: disallow question mark in slug
- [ ] strip it out frontend
- [ ] put `pattern` on the input
- [ ] strip it out backend

### PR DESCRIPTION
**Pull request by Haroenv**
- [x] filter frontend
- [x] validate
- [x] put a `pattern` on the input

fixes #68

### CODE REVIEW

File: src/main/java/com/pollistics/models/validators/PollValidator.java [hunk 1]
```diff
@@ -16,6 +17,10 @@ public boolean supports(Class<?> clazz) {
 	public void validate(Object target, Errors errors) {
 		Poll poll = (Poll) target;
 
+		if (Pattern.matches("\\?|\\.|\\s|#|\\/|:", poll.getSlug())) {
```
Comment (Haroenv) [inline src/main/java/com/pollistics/models/validators/PollValidator.java]: This doesn't reject the input for some reason...

File: src/main/java/com/pollistics/models/validators/PollValidator.java [hunk 2]
```diff
@@ -16,6 +18,10 @@ public boolean supports(Class<?> clazz) {
 	public void validate(Object target, Errors errors) {
 		Poll poll = (Poll) target;
 
+		if (Pattern.matches("\\?|\\.|\\s|#|/|:", poll.getSlug())) {
```
Comment (Haroenv) [inline src/main/java/com/pollistics/models/validators/PollValidator.java]: Still failing 😕

**Review by hannnegoossens — APPROVED** [review]
proud of our scrum master!

**Review by Haroenv — COMMENTED** [review]
Very proud of myself. It does the validation on three levels

**Pull request merged**